modules/v2x/v2x_proxy/os_interface/os_interface.cc

```diff
@@ -58,7 +58,7 @@ bool OsInterFace::InitReaders() {
         }
         cond_planning_.notify_one();
       });
-  return !!localization_reader_ && !!planning_reader_;
+  return nullptr != localization_reader_ && nullptr != planning_reader_;
```

Comment:
Why not `return localization_reader_ && planning_reader_;`

Reply:
To emphasize that these are two pointers, not two boolean values. ```&&``` should **ONLY** be used to connect boolean values